src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,19 @@
+@import "./../../utils/variables.scss";
+
+.TodoInfo {
+  width: max-content;
+  padding: 8px;
+  margin: 20px 0;
+  border-radius: $-border-radius;
+  background-color: $-color-background-todo;
+
+  &__title {
+    margin: 4px 0;
+    font-size: inherit;
```

Comment:
Consider using a more specific font size instead of `inherit`. For example, `font-size: 16px;`. 📏